docs/user-guide/configure.md

```diff
@@ -658,3 +658,37 @@ For example:
 > This config option should not be overridden on a per-file basis.
 
 [More info](options.md#fix).
+
+## `formatter`
+
+Specify the formatter to format your results.
+
+Options are:
+
+- The name of a provided formatter.
+
+```json
+{
+  "formatter": "string"
+}
+```
+
+- A path to a custom formatter function.
+
+```json
+{
+  "formatter": "path/to/customformatter.js"
+}
+```
+
+- A formatter function
+
+```js
+export default {
+  formatter: () => {
+    // ...
+  }
+};
+```
```

Comment:
[suggestion] Thanks for improving the document. Additionally, can you make one list by indenting the code blocks? E.g.

````suggestion
- The name of a provided formatter.
  ```json
  {
    "formatter": "string"
  }
  ```
- A path to a custom formatter function.
  ```json
  {
    "formatter": "path/to/customformatter.js"
  }
  ```
- A formatter function.
  ```js
  export default {
    formatter: () => { /*..*/ }
  };
  ```
````